From 2ad0a4b8c3852ad937fec9fc85780230257c821e Mon Sep 17 00:00:00 2001
From: Julian Brost <julian.brost@icinga.com>
Date: Wed, 17 Nov 2021 16:11:15 +0100
Subject: [PATCH] Add missing include to fix non-unity builds
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This commit fixes the following build error:

    [ 55%] Building CXX object lib/icinga/CMakeFiles/icinga.dir/usergroup.cpp.o
    lib/icinga/usergroup.cpp:79:24: error: incomplete type ‘icinga::Notification’ used in nested name specifier
       79 | std::set<Notification::Ptr> UserGroup::GetNotifications() const
          |                        ^~~
---
 lib/icinga/usergroup.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/icinga/usergroup.cpp b/lib/icinga/usergroup.cpp
index 6ff4b89d42..49bd7186ac 100644
--- a/lib/icinga/usergroup.cpp
+++ b/lib/icinga/usergroup.cpp
@@ -2,6 +2,7 @@

 #include "icinga/usergroup.hpp"
 #include "icinga/usergroup-ti.cpp"
+#include "icinga/notification.hpp"
 #include "config/objectrule.hpp"
 #include "config/configitem.hpp"
 #include "base/configtype.hpp"
